packages/terra-clinical-label-value-view/lib/LabelValueView.scss

```diff
@@ -0,0 +1,17 @@
+@import '~terra-mixins';
+
+.terraClinical-LabelValueView {
+  border: 0;
+  padding: 0.1785rem;
+}
+
+.terraClinical-LabelValueView-label {
+  color: $terra-grey-50;
+  font-size: 1rem;
+  line-height: 20px;
```

Comment:
Recently we had a discussion to keep line-heights unit-less (dropping px / rem): https://css-tricks.com/almanac/properties/l/line-height/#article-header-id-0 

This line-height value is also different than the one used below